src/Domains/Domains.jl

```diff
@@ -66,6 +70,50 @@ end
 IntervalDomain(coords::ClosedInterval; kwargs...) =
     IntervalDomain(coords.left, coords.right; kwargs...)
 
+
+function XIntervalDomain(;
```

Comment:
@Sbozzolo what do you think about adding doc strings to all of these convenience constructors, so that users know what the kwargs are? I think just copying the code itself into the doc string should be pretty self explanatory, and it'll be easy to keep up-to-date.